src/api/common/schemas/natural.json

```diff
@@ -0,0 +1,7 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "title": "natural",
+  "description": "A non-negative integer",
+  "type": "integer",
+  "minimum": 0
```

Comment:
I think the first ledger sequence is 1, so, really we want positive integers right?